### ISSUE TITLE
Markdown highlight not showing when treesitter is enabled.

### ISSUE THREAD

**Issue by jerry-peng** [maxmx03/solarized.nvim#64]
Noticed Markdown syntax highlight is not displayed at all after updating markdown treesitter parser. This is likely caused by treesitter's recently-merged breaking change to align standard capture names between treesitter and helix.
- Tracking issue: https://github.com/nvim-treesitter/nvim-treesitter/issues/4799
- Commit: https://github.com/nvim-treesitter/nvim-treesitter/commit/1ae9b0e4558fe7868f8cda2db65239cfb14836d0

The capture names need to be updated, specifically (copied from commit above):
```
* tree-sitter "standard capture names"
  (https://github.com/tree-sitter/tree-sitter/blob/3f44b896852eb7daaa6df4fb778c9bb52c70c815/highlight/src/lib.rs#L20-L72):

  - `@parameter` -> `@variable.parameter`
  - `@field` -> `@variable.member`
  - `@namespace` -> `@module`
  - `@float` -> `@number.float`
  - `@symbol` -> `@string.special.symbol`
  - `@string.regex` -> `@string.regexp`
  - `@text.*` -> `@markup.*` (`strong`, `italic`, `link`, `strikethrough`; with exceptions; see below)
  - `@text.title` -> `@markup.heading`
  - `@text.literal` -> `@markup.raw`
  - `@text.reference` -> `@markup.link`
  - `@text.uri` -> `@markup.link.url` (in markup links)
  - `@string.special` -> `@markup.link.label` (non-url links)
  - `@punctuation.special` -> `@markup.list` (markdown lists only; move subitems from `@text.todo`)

* Helix captures
  (https://docs.helix-editor.com/master/themes.html#syntax-highlighting):

  - `@method` -> `@function.method`
  - `@method.call` -> `@function.method.call`
  - `@text.{todo,warning,note,danger}` -> `@comment.{error,warning,hint,info,todo}`
  - `@text.diff.{add,delete,}` -> `@diff.{plus,minus,delta}`
  - `@text.uri` -> `@string.special.url` (outside markup)
  - `@preproc` -> `@keyword.directive`
  - `@define` -> `@keyword.directive`(`.define`?)
  - `@storageclass` -> `@keyword.storage`
  - `@conditional` -> `@keyword.conditional`
  - `@debug` -> `@keyword.debug`
  - `@exception` -> `@keyword.exception`
  - `@include` -> `@keyword.import`
  - `@repeat` -> `@keyword.repeat`
```

I can spend some time and put up a PR if you prefer @maxmx03

**Comment by jerry-peng** [comment]
PR: https://github.com/maxmx03/solarized.nvim/pull/66

**Comment by jerry-peng** [comment]
There are many deprecated capture names that should be removed. I'll push another PR once #66 is merged
